libs/pageserver_api/src/models.rs

```diff
@@ -123,9 +123,15 @@ pub struct TenantInfo {
     pub has_in_progress_downloads: Option<bool>,
 }
 
+/// This represents the output of the "timeline_detail" and "timeline_list" API calls.
 #[serde_as]
 #[derive(Debug, Serialize, Deserialize, Clone)]
-pub struct LocalTimelineInfo {
+pub struct TimelineInfo {
+    #[serde_as(as = "DisplayFromStr")]
+    pub tenant_id: TenantId,
```

Comment:
That would require changing https://github.com/neondatabase/neon/blob/9fe4548e13774d7f1e5f9b5d23e57da971419442/pageserver/src/http/openapi_spec.yml#L610-L625 and https://github.com/neondatabase/cloud/blob/a90901d4269b3fc1cbec1286f197ee07e70923f0/swagger/pageserver.yml#L558-L573

And the latter would affect the console:

https://github.com/neondatabase/cloud/blob/a90901d4269b3fc1cbec1286f197ee07e70923f0/goapp/internal/client/psclient/psclient.go#L86-L91

https://github.com/neondatabase/cloud/blob/a90901d4269b3fc1cbec1286f197ee07e70923f0/goapp/internal/operationsv2/provisioner_common.go#L588-L603

I think we can incorporate the changes in forward-compatible way, but need to discuss it further maybe.

Reply:
Good catch, I didn't realize the control plane uses these. Obvious in hindsight..

I kept little stub versions of 'local' and 'remote' fields, for backwards-compatibility, that duplicate the fields that the control plane uses. We can remove them later, when the control plane has been updated. (But I don't want to update the control plane until the new pageserver version has been deployed all the way to production, and we're sure we don't need to roll back anymore).